azure-key-vault/info.json

```diff
@@ -0,0 +1,1038 @@
+        "title": "Application (client) ID",
+        "name": "client_id",
+        "type": "text",
+        "description": "Unique ID of the Azure Active Directory application that is used to create an authentication token required to access the API.",
+        "tooltip": "Unique ID of the Azure Active Directory application that is used to create an authentication token required to access the API.",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "value": ""
+      },
+      {
+        "title": "Application (Client) Secret",
+        "name": "client_secret",
+        "type": "password",
+        "description": "Unique Client Secret of the Azure Active Directory application that is used to create an authentication token required to access the API. For information on how to get the secret key, see https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/exposed-apis-create-app-webapp.",
+        "tooltip": "Unique Client Secret of the Azure Active Directory application that is used to create an authentication token required to access the API.",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "value": ""
+      },
+      {
+        "title": "Verify SSL",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "name": "verify_ssl",
+        "value": true,
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "list_key_vault",
+      "title": "List Key Vaults",
+      "description": "The List operation gets information about the vaults associated with the subscription.",
+      "category": "investigation",
+      "annotation": "list_key_vault",
+      "output_schema": {
+        "value": [
+          {
+            "id": "",
+            "name": "",
+            "type": "",
+            "location": "",
+            "tags": {}
+          }
+        ],
+        "nextLink": ""
+      },
+      "parameters": [
+        {
+          "title": "Skip Token",
+          "name": "skip_token",
+          "type": "text",
+          "description": "(Optional) Skip Token is only used if a previous operation returned a partial result. If a previous response contains a nextLink element, the value of the nextLink element will include a skiptoken parameter that specifies a starting point to use for subsequent calls.",
+          "tooltip": "Skip Token is only used if a previous operation returned a partial result. If a previous response contains a nextLink element, the value of the nextLink element will include a skiptoken parameter that specifies a starting point to use for subsequent calls.",
+          "required": false,
+          "editable": true,
+          "visible": true
+        },
+        {
+          "title": "Size",
```

Comment:
Is there any default value? If Yes, please keep as default value.

Reply:
There is no default value

Reply:
No default value mentioned in documentation.